care/facility/summarisation/patient_summary.py

```diff
@@ -0,0 +1,112 @@
+from datetime import timedelta
+
+from django.db.models import Q,Subquery
+from django.utils.timezone import now
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import cache_page
+from django.views.decorators.vary import vary_on_cookie
+from rest_framework import serializers, status, viewsets
+from rest_framework.generics import get_object_or_404
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
+from rest_framework.permissions import IsAuthenticated
```

Comment:
Codacy found an issue: [Unable to import 'rest_framework.permissions'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5789522)